src/identify/index.js

```diff
@@ -290,7 +264,7 @@ class IdentifyService {
    *
    * @private
    * @param {object} options
-   * @param {*} options.stream
+   * @param {DuplexIterable} options.stream
```

Comment:
We should probably track adding a type for `Stream` in the interfaces module, that should probably live with Connection. While any `DuplexIterable` should be fine, we really want these to be Stream's